### PR TITLE
Family Private Law - citizen frontend environment setup

### PR DESCRIPTION
**Pull request by AlokDatta**
### Change description ###
prl-citizen-frontend image policy setup
prl-citizen-frontend perftest and ithc environment flux setup
prl-cos ithc environment flux setup
prl-dgs ithc environment flux setup

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/private-law/prl-cos/ithc.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+kind: HelmRelease
+metadata:
+  name: prl-cos
+  namespace: private-law
+spec:
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/prl/cos:prod-4bccd66-20220516112343 #{"$imagepolicy": "flux-system:prl-cos"}
```
Comment (reespozzi) [inline k8s/namespaces/private-law/prl-cos/ithc.yaml:11]: `image:` not needed in any of the {env}.yaml files if it's just using the base image from the prl-cos.yaml file

File: k8s/namespaces/private-law/prl-dgs/ithc.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+kind: HelmRelease
+metadata:
+  name: prl-dgs
+  namespace: private-law
+spec:
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/prl/dgs:prod-710d3aa-20220516110533 #{"$imagepolicy": "flux-system:prl-dgs"}
```
Comment (reespozzi) [inline k8s/namespaces/private-law/prl-dgs/ithc.yaml:11]: Same here

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ spec:
     path: stable/prl-citizen-frontend
   values:
     java:
-      image: hmctspublic.azurecr.io/prl/cos:latest #{"$imagepolicy": "flux-system:prl-citizen-frontend"}
+      image: hmctspublic.azurecr.io/prl/cos:latest #{"$imagepolicy": "flux-system:demo-prl-citizen-frontend"}
```
Comment (reespozzi) [inline k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml:17]: Can you use a specific image here if possible

**Review by reespozzi — CHANGES_REQUESTED** [review]

File: k8s/namespaces/private-law/prl-cos/ithc.yaml (hunk 1 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-cos/ithc.yaml]: Addressed and removed as it is there with the prl-cos.yaml file

File: k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml (hunk 3 above)
Comment (AlokDatta) [reply k8s/namespaces/private-law/prl-citizen-frontend/demo.yaml]: Updated with PR image.

**Review by cshiwarale — APPROVED** [review]

**Pull request merged**